.bazelrc

```diff
@@ -105,6 +104,7 @@ build:rbe-toolchain-clang-libc++ --config=rbe-toolchain
 build:rbe-toolchain-clang-libc++ --crosstool_top=@rbe_ubuntu_clang_libcxx//cc:toolchain
 build:rbe-toolchain-clang-libc++ --extra_toolchains=@rbe_ubuntu_clang_libcxx//config:cc-toolchain
 build:rbe-toolchain-clang-libc++ --action_env=CC=clang --action_env=CXX=clang++ --action_env=PATH=/usr/sbin:/usr/bin:/sbin:/bin:/usr/lib/llvm-8/bin
+build:rbe-toolchain-clang-libc++ --define force_libcpp=enabled
```

Comment:
Is this change unrelated?

Reply:
It is related, we didn't need this before because toolchain handled all `envoy_select_force_libcpp` cases but it is no longer true (needed for selecting dynamic stdlib now)

Reply:
I have no idea, I just noticed that it was removed here and I wanted to make sure you meant to do it. Seems fine to remove.